capstone-ui/src/Components/App/App.jsx

```diff
@@ -70,7 +71,7 @@ export default function App() {
             </div>
             <div className="content">
               <Routes>
-                <Route path='/' element={<p>Message</p>}/>
+                <Route path='/' element={ user ? (<Feed loggedin={true}/>) : (<Feed loggedin={false}/>)}/>
```

Comment:
Would it be easier to provide  user object instead? This way in future you can use User data if needed in Feed.

Reply:
I see what you mean, and I could do that, but with UserContext I'll be able to use the user inside the feed with no issues.